config/deployer/config.ts

```diff
@@ -1379,3 +1344,30 @@ export const addLiquidity = async (config: Config) => {
     console.log(chalk.red(`\n    ✖ Failed to add liquidity: `) + chalk.gray(e) + "\n");
   }
 };
+
+export const nodeReadConfig = async (chain: Chain) => {
```

Comment:
is there a reason why you use this instead of the previous getConfigFromNetwork ? 

i believe we had issues with vercel deployment if we tried to load the config dynamically